### DISCUSSION TITLE
Getting issue or error while running the .env file

### DISCUSSION THREAD

**Discussion by PratikKhaire** [Cyfrin/foundry-full-course-cu#287] [Q&A] [answered]
Description 

G**etting error message while run the command to run the script for compiling the smart contract** 

i am created .env file and added the private key and rpc_url to it 
after doing this i want to run this script using the command 
```terminal
forge script script/DeploySimpleStorage.s.sol --rpc-url $RPC_URL --broadcast --private-key $PRIVATE_KEY
```
But i am getting error while ruining this command and error is here 
```error
Error: 
error sending request for url (https://127.0.0.1:8545/): error trying to connect: received corrupt message of type InvalidContentType

Context:
- Error #0: error trying to connect: received corrupt message of type InvalidContentType
- Error #1: received corrupt message of type InvalidContentType
```
also i already run the anvil and all required stuff that are show in video but get that error try to help me

**Comment by f3d0ss** [comment]
It's probably the `https` instead of `http` in the `RPC_URL` enviroment variable

**Comment by ChefAharoni** [comment, marked as answer]
> It's probably the `https` instead of `http` in the `RPC_URL` enviroment variable

What @f3d0ss said and/or - it looks like an Anvil URL, have you started Anvil in another terminal window?
Anvil must be running in order to perform tasks on it.

**Comment by PratikKhaire** [comment]
@ChefAharoni ok

**Reply by ChefAharoni** [reply]
Has it worked?
